File: ulyp-ui/src/main/java/com/ulyp/ui/code/find/GradleJarFile.java
package com.ulyp.ui.code.find;

import java.io.File;
import java.nio.file.Path;

public class GradleJarFile extends JarFile {

    protected GradleJarFile(File path) {
        super(path);
    }

    protected static boolean isGradleJarFile(Path path) {
        return path.toString().contains(".gradle");
    }
}

File: ulyp-ui/src/main/kotlin/com/ulyp/ui/code/find/GradleJarFile.kt
package com.ulyp.ui.code.find

import java.io.File
import java.nio.file.Path

class GradleJarFile protected constructor(path: File) : JarFile(path) {
    companion object {
        protected fun isGradleJarFile(path: Path): Boolean {
            return path.toString().contains(".gradle")
        }
    }
}